### DISCUSSION TITLE
corseplay does not work on three screens after the last update 7.4.1.0 Resolution 5760x1080

### DISCUSSION THREAD

**Discussion by Caltec2020** [Courseplay/Courseplay_FS22#3137] [General]
corseplay does not work on three screens after the last update 7.4.1.0 Resolution 5760x1080

**Comment by Tensuko** [comment]
@schwiti6190 guess that was the UI improvement as well

**Comment by Tensuko** [comment]
Pls try this pull request if it does work for you.
https://github.com/Courseplay/Courseplay_FS22/pull/3143

**Comment by Tensuko** [comment]
Can you pls try this Version?
[FS22_Courseplay.zip](https://github.com/Courseplay/Courseplay_FS22/files/14618997/FS22_Courseplay.zip)

**Reply by Caltec2020** [reply]
Thanks Tensuko,
The link you sent me allows me to see the menu when I make a Courseplay.
The menu is on the far left, in the previous version it was centered on the middle of my 3 screens. Which was more convenient. But now it works.

**Reply by Tensuko** [reply]
We have to keep it like that, if it does work.
Experimenting with moving it to another position could other resolution to get new issues.
Will be part of the next Release then.

**Reply by Caltec2020** [reply]
I understand Thank you
Courseplay is the most important mod, together with AutoDrive is the necessary mod if, like me, you play alone on large maps.
I sometimes drive with 10+ helpers. But it goes a little beyond the graphics. This can affect the image quality.

**Reply by Tensuko** [reply]
Can't change the performance drop.
To many calculations that are going on but 10+ Helper are already a lot.
I know other have more Helper running but thats even more performance lost.
